service/dap/server.go

```diff
@@ -1898,47 +1898,50 @@ func (s *Server) childrenToDAPVariables(v *fullyQualifiedVariable) ([]dap.Variab
 
 func getNamedVariableCount(v *proc.Variable) int {
 	namedVars := 0
-	if len(v.Children) > 0 && (v.Kind == reflect.Array || v.Kind == reflect.Slice) {
-		childKind := v.Children[0].RealType.Common().ReflectKind
-		if len(v.Children) > 0 && (childKind == reflect.Uint8 || childKind == reflect.Int32) {
-			// string value of array/slice of bytes and runes.
-			namedVars += 1
-		}
+	if canConvertToString(v) {
```

Comment:
Well, many things can be converted to string. How about isStringEquivalent? Or even just the explicit IsListOfBytesOrRunes

Reply:
Done.